app/views/layouts/_content.html.haml

```diff
@@ -63,6 +63,9 @@
             = render :partial => "layouts/listnav"
           = yield :left
         #custom_left_cell
+- elsif @angular_app_name
+  .container-fluid{:style => "overflow: hidden; height: 100%;", 'data-ng-app' => "#{@angular_app_name}"}
+    %div{'ui-view' => ''}
 - else
```

Comment:
I'll wait for your PR @AparnaKarve to be merged as well, and build on top of that. I think separate place for enabling angular is better approach, since it is much clearer to understand what is happening.

About directing to non-angular pages, that's axactly what I did. I also created couple  of redirects in [ems_middleware_controller.rb](https://github.com/ManageIQ/manageiq/pull/7623/files#diff-0b135735040fdab48c8d6d20a18e896fR32) and then deal with these anchors in [providers_module.js](https://github.com/ManageIQ/manageiq/pull/7623/files#diff-adc7c380f444a16aede45b0d13ca58d3R18) so users can go to these urls wihtout error from ruby router. I think that this is good, since when user is in angularized part of application ui-router is taking care of this and no long gets for resources.

We should discuss however if we want to use ui-router or stick to angular router, I'd prefere ui-router, since it is much robust and stronger, but it's additional file file which is loaded so that might be problem. But I am okay with either way.